easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -624,15 +635,11 @@ def configure_step(self):
 
         # creates log entries for python being used, for debugging
         run_cmd("%s -V" % self.python_cmd, verbose=False, trace=False)
```

Comment:
Why not combine the `python -V` too, and make it a single command?

```python
cmd = '; '.join([
    "%s -V" % self.python_cmd,
    "%s -c 'import sys; print(sys.executable, sys.path)'" % self.python_cmd,
])
run_cmd(cmd, verbose=False, trace=False)

Reply:
Good idea. Done